<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>808</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLabel" name="field">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>148</y>
     <width>601</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image tiles/field2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_adv">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>198</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_pv">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>351</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_XP">
   <property name="geometry">
    <rect>
     <x>746</x>
     <y>382</y>
     <width>126</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="nom_adv">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>174</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>American Typewriter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pokemon</string>
   </property>
  </widget>
  <widget class="QLabel" name="nom_adv_2">
   <property name="geometry">
    <rect>
     <x>717</x>
     <y>331</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>American Typewriter</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pokemon</string>
   </property>
  </widget>
<widget class="QTextBrowser" name="dialogue">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>418</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
</widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>607</x>
     <y>175</y>
     <width>19</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>334</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>23</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>267</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image tiles/pokemon_Combat/back/2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>208</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image tiles/pokemon_Combat/front/2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>508</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Downloads/1000_F_189138228_Khn6fbgC2zaTi90hsnTZqFc3MHPcQVib.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>697</x>
     <y>708</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>POKEMON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>528</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ATTAQUE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>687</x>
     <y>528</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ATTAQUE SP2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>477</x>
     <y>708</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>FUITE</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cadre">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>0</y>
     <width>1609</width>
     <height>920</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image tiles/cadre.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
